### PR TITLE
samples: wifi: Update command output and help for power save

### PR DESCRIPTION
**Pull request by rado17**
Update help text to provide information for TWT and power save commands in wifi shell.

Signed-off-by: Ravi Dondaputi <ravi.dondaputi@nordicsemi.no>

### CODE REVIEW

**Review by sachinthegreen — APPROVED** [review]

**Comment by sachinthegreen** [comment]
Please add reference to the Jira bug which this PR solves.

File: samples/wifi/shell/README.rst [hunk 1]
```diff
@@ -89,6 +89,28 @@ Supported CLI commands
...
+   * - ps
+     - | Configure power save
+       | No argument: Prints current configuration
+       | on - Turns on power save
+       | off - Turns off power save
+   * - twt
+     - | Manage TWT flows
+       |
+       | Subcommands:
+       |
```
Comment (richabp) [inline samples/wifi/shell/README.rst:101]:
```suggestion
```

File: samples/wifi/shell/README.rst [hunk 2]
```diff
@@ -89,6 +89,28 @@ Supported CLI commands
...
+       | setup         : Start a TWT flow:
+       |          <negotiation_type, 0: Individual, 1: Broadcast, 2: Wake TBTT>
+       |          <setup_cmd: 0: Request, 1: Suggest, 2: Demand>
```
Comment (richabp) [inline samples/wifi/shell/README.rst:104]:
```suggestion
       | setup - Start a TWT flow:
       |          <negotiation_type: 0 - Individual, 1 - Broadcast, 2 - Wake TBTT>
       |          <setup_cmd: 0 - Request, 1 - Suggest, 2 - Demand>
```

File: samples/wifi/shell/README.rst [hunk 3]
```diff
@@ -89,6 +89,28 @@ Supported CLI commands
...
+       |          <dialog_token> <flow_id> <responder> <trigger> <implicit>
+       |          <announce> <twt_wake_interval_ms> <twt_interval_ms>
+       |
+       | teardown      : Teardown a TWT flow:
+       |          <negotiation_type, 0: Individual, 1: Broadcast, 2: Wake TBTT>
+       |          <setup_cmd: 0: Request, 1: Suggest, 2: Demand>
```
Comment (richabp) [inline samples/wifi/shell/README.rst:110]:
```suggestion
       | teardown - Teardown a TWT flow:
       |            <negotiation_type: 0 - Individual, 1 - Broadcast, 2 - Wake TBTT>
       |            <setup_cmd: 0 - Request, 1 - Suggest, 2 - Demand>
```

**Review by richabp — CHANGES_REQUESTED** [review]

**Review by richabp — CHANGES_REQUESTED** [review]

File: samples/wifi/shell/README.rst [hunk 4]
```diff
@@ -89,6 +89,28 @@ Supported CLI commands
      - Configure the Wi-Fi interface as access point mode
    * - ap_disable
      - Configure the Wi-Fi interface as station mode
+   * - ps
+     - | Configure power save
+       | No argument: Prints current configuration
+       | on - Turns on power save
+       | off - Turns off power save
```
[the remark this answers is not in the document]
Comment (rado17) [reply samples/wifi/shell/README.rst:96]: Options mentioned here are in small-case and the suggested options are not accepted by the command.

[the remark this answers is not in the document]
Comment (rado17) [reply samples/wifi/shell/README.rst]: Included other suggested changes.

File: samples/wifi/shell/README.rst (hunk 1 above)
Comment (rado17) [reply samples/wifi/shell/README.rst]: Removed the suggested lines.

File: samples/wifi/shell/README.rst [hunk 5]
```diff
@@ -89,6 +89,28 @@ Supported CLI commands
...
+   * - twt
+     - | Manage TWT flows
```
[the remark this answers is not in the document]
Comment (rado17) [reply samples/wifi/shell/README.rst]: Done

File: samples/wifi/shell/README.rst (hunk 2 above)
Comment (rado17) [reply samples/wifi/shell/README.rst]: Done,

File: samples/wifi/shell/README.rst (hunk 3 above)
Comment (rado17) [reply samples/wifi/shell/README.rst]: Done

File: samples/wifi/shell/README.rst [hunk 6]
```diff
@@ -89,6 +89,28 @@ Supported CLI commands
...
+       |          <setup_cmd: 0: Request, 1: Suggest, 2: Demand>
+       |          <dialog_token> <flow_id> <responder> <trigger> <implicit>
+       |          <announce> <twt_wake_interval_ms> <twt_interval_ms>
+       |
+       | teardown      : Teardown a TWT flow:
+       |          <negotiation_type, 0: Individual, 1: Broadcast, 2: Wake TBTT>
+       |          <setup_cmd: 0: Request, 1: Suggest, 2: Demand>
+       |          <dialog_token> <flow_id>
+       |
+       | teardown_all  : Teardown all TWT flows
```
[the remark this answers is not in the document]
Comment (rado17) [reply samples/wifi/shell/README.rst]: Done.

**Review by richabp — APPROVED** [review]

**Review by nordicjm — CHANGES_REQUESTED** [review]
Marking as -1 because of doc-required tag and no doc approval and no one from doc team in review list

**Comment by rado17** [comment]
> Marking as -1 because of doc-required tag and no doc approval and no one from doc team in review list

It has been approved by @richabp.  Do we need to tag anybody else?

**Comment by nordicjm** [comment]
> It has been approved by @richabp. Do we need to tag anybody else?

Someone in https://github.com/orgs/nrfconnect/teams/ncs-doc-owners/members

**Review by nordicjm — APPROVED** [review]

**Pull request merged**